apps/src/JsDebuggerUi.js

```diff
@@ -380,6 +434,30 @@ JsDebuggerUi.prototype.onMouseMoveDebugResizeBar = function (event) {
   utils.fireResizeEvent();
 };
 
+/**
+ *  Handle mouse moves while dragging the debug resize bar.
+ */
+JsDebuggerUi.prototype.onMouseMoveWatchersResizeBar = function (event) {
+  const watchersResizeBar = document.getElementById('watchersResizeBar');
+  const watchersDiv = document.getElementById('debug-watch');
+  const watchersHeaderDiv = document.getElementById('debug-watch-header');
+  const debugConsoleDiv = document.getElementById('debug-console');
```

Comment:
We grab this same set of DOM elements in `onResetWatchersResizeBar` above.  Maybe a good helper to extract?  Better yet, can we cache references to these elements so we're not doing DOM lookups inside our mousemove handler?